homeassistant/components/qvr_pro/__init__.py

```diff
@@ -49,10 +51,11 @@ def setup(hass, config):
     user = conf[CONF_USERNAME]
     password = conf[CONF_PASSWORD]
     host = conf[CONF_HOST]
+    port = conf[CONF_PORT]
```

Comment:
It looks like port is optional without default in the config schema. Then we need to use `dict.get`.
```suggestion
    port = conf.get(CONF_PORT)
```
Or put a default in the schema.